WcaOnRails/db/migrate/20161227202950_add_preferred_locale_to_users.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class AddPreferredLocaleToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :preferred_locale, :string, default: 'en'
```

Comment:
I don't like the default. It makes it impossible to tell apart people who have **chosen** English from those who have just defaulted to it.